content/applications/inventory_and_mrp/barcode/setup/operation_types.rst

```diff
@@ -0,0 +1,216 @@
+   picking step.
+
+.. important::
+   Printing barcodes, by default, will either directly download a PDF file of the selected barcodes
+   or open a new tab with a downloadable or printable PDF. To turn this into a direct printing
+   operation, :doc:`connect a printer <../../../general/iot/devices/printer>`.
+
+When first opening Barcode
+--------------------------
+
+When first opening the **Barcode** app, there is a prompt with options to print barcodes for
+commands and operations, as well as some sample barcodes to print and use for testing. Clicking
+these links is the easiest way to print command and operation barcodes, but will no longer be
+displayed after the first time it is clicked.
+
+.. image:: operation_types/barcode-initial-print-prompt.png
+   :alt: A prompt to print demo data or barcodes for operations.
+
+From Inventory settings
+-----------------------
+
+If the barcode print prompt is not available, inventory commands and operation types barcodes can
+always be printed from the *Settings* page. Go to :menuselection:`Inventory app --> Configuration
+--> Settings` and in the :guilabel:`Barcode` section under the :guilabel:`Barcode Scanner` setting,
+click :icon:`fa-print` :guilabel:`Print barcode commands and operation types`.
+
+.. image:: operation_types/print-operations-and-commands.png
+   :alt: Link to print inventory operations and commands.
+
+Individual operations
+---------------------
+
+Barcodes for individual operations, like receipts and manufacturing orders, can also be printed from
+the *Operations Types* configuration menu. To select operations to print, go to
+:menuselection:`Inventory app --> Configuration --> Operations Types` and tick the box by each
+operation type required.
+
+.. image:: operation_types/barcode-print-operations.png
+   :alt: Available operations selected making a 'Print' button appear.
+
+Next, click the :icon:`fa-print` :guilabel:`Print` button and select either :guilabel:`Operation
+type (PDF)` to download a PDF with the operation barcodes, or :guilabel:`Operation type (ZPL)` to
+send the print job to a :ref:`ZPL-enabled printer <iot/link-printer>`.
+
+.. note::
+   The :icon:`fa-print` :guilabel:`Print` button does not appear until at least one operation type
+   is selected.
+
+.. _inventory/barcode/print_order:
+
+Print picking operations for specific orders
+============================================
+
+Receipts, manufacturing orders, transfers, and delivery orders can all be printed in advance and are
+referred to in the print options menu as *picking operations* or *picking*.
+
+In Barcode
+----------
+
+From the **Barcode** app, tap the :guilabel:`Operations` button, select the operation type, and then
+the specific order to print. From there, either scan the `PRINT PICKING OPERATION` barcode if there
+is one available, or tap the :icon:`fa-cog` `(gear)` icon to open the **Barcode** actions menu and
+tap the :guilabel:`Print Picking Operation` button.
```

Comment:
Please be mindful when [using find-and-replace regex](https://github.com/odoo/documentation/pull/13355#discussion_r2105199063), this still needs a guilabel
```suggestion
is one available, or tap the :icon:`fa-cog` :guilabel:`(gear)` icon to open the **Barcode** actions
menu and tap the :guilabel:`Print Picking Operation` button.
```